commit/report.go

```diff
@@ -112,8 +114,30 @@ func (p *Plugin) Reports(
 func (p *Plugin) ShouldAcceptAttestedReport(
 	ctx context.Context, seqNr uint64, r ocr3types.ReportWithInfo[[]byte],
 ) (bool, error) {
-	if isActive, err := p.checkActiveInstance(ctx); err != nil || !isActive {
-		return false, err
+	// check if we support the dest, if not we can't do the checks needed.
+	supports, err := p.chainSupport.SupportsDestChain(p.oracleID)
+	if err != nil {
+		return false, fmt.Errorf("supports dest chain: %w", err)
+	}
+
+	if !supports {
+		p.lggr.Warnw("dest chain not supported, can't run report acceptance procedures")
+
+		// TODO: return false here or a non-nil error?
+		return false, nil
```

Comment:
It seems like libocr handles a returned `false` and a non-nil error the same way: https://github.com/smartcontractkit/libocr/blob/adbe57025f12b9958907bb203acba14360bf8ed2/offchainreporting2plus/internal/ocr3/protocol/transmission.go#L118-L145

Leaning towards just returning false since this isn't really an error strictly speaking.